buildtest/config_schema.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$id": "https://raw.githubusercontent.com/HPC-buildtest/buildtest-framework/devel/buildtest/config_schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Buildtest Configuration Schema Definition",
+    "type": "object",
+    "defs": {
```

Comment:
Are any properties required (they would go up here).

Reply:
Yes so far there are only two required properties `editor` and `queues` see https://github.com/shahzebsiddiqui/buildtest-framework/blob/configuration_jsonschema/buildtest/settings/settings.schema.json#L54

Is it best practice to put `required` above the `object` field. When i was learning the json schema https://json-schema.org/understanding-json-schema/reference/object.html#required-properties i saw the `required` key defined at bottom.

Reply:
I just missed it, the order is fairly trivial! I've also seen it at the top, as more of a "heads up this is required." However, I'm indifferent to order, bottom is fine too.